Assets/Tests/InputSystem/CoreTests_Actions_Rebinding.cs

```diff
@@ -1332,4 +1332,133 @@ public void Actions_CanSaveAndLoadRebinds()
+    {
+        var actionMap = new InputActionMap();
+        var firstActionInMap = actionMap.AddAction("action1");
+        var secondActionInMap = actionMap.AddAction("action2");
+
+        actionMap.AddBinding("<Keyboard>/a", firstActionInMap);  // first, 0
+        actionMap.AddBinding("<Keyboard>/b", secondActionInMap); // second, 0
+        actionMap.AddBinding("<Keyboard>/c", firstActionInMap);  // first, 1
+        actionMap.AddBinding("<Keyboard>/d", secondActionInMap); // second, 1
+
+        Assert.AreEqual(0, actionMap.FindBindingRelativeToMap(new InputBinding())); // unconditional
+    }
+
+    [Test]
+    [Category("Actions")]
+    public void Actions_ActionMapFindBindingRelativeToMap_ShouldReturnIndexOfBinding_IfMatches()
+    {
+        var actionMap = new InputActionMap();
+        var firstActionInMap = actionMap.AddAction("action1");
+        var secondActionInMap = actionMap.AddAction("action2");
+
+        actionMap.AddBinding("<Keyboard>/a", firstActionInMap);  // first, 0
+        actionMap.AddBinding("<Keyboard>/b", secondActionInMap); // second, 0
+        actionMap.AddBinding("<Keyboard>/c", firstActionInMap);  // first, 1
+        actionMap.AddBinding("<Keyboard>/d", secondActionInMap); // second, 1
+        actionMap.AddBinding("<Keyboard>/e", secondActionInMap); // second, 2
+
+        Assert.AreEqual(0, actionMap.FindBindingRelativeToMap(new InputBinding("<Keyboard>/a"))); // exact match
+        Assert.AreEqual(1, actionMap.FindBindingRelativeToMap(new InputBinding("<Keyboard>/b"))); // exact match
+        Assert.AreEqual(2, actionMap.FindBindingRelativeToMap(new InputBinding("<Keyboard>/c"))); // exact match
+        Assert.AreEqual(3, actionMap.FindBindingRelativeToMap(new InputBinding("<Keyboard>/d"))); // exact match
+        Assert.AreEqual(4, actionMap.FindBindingRelativeToMap(new InputBinding("<Keyboard>/e"))); // exact match
+    }
+
+    [Test]
+    [Category("Actions")]
+    public void Actions_InputActionSetupExtensionsChange_ShouldChangeBindingIfFound()
+    {
+        var actionMap = new InputActionMap();
+        var firstActionInMap = actionMap.AddAction("action1");
+        var secondActionInMap = actionMap.AddAction("action2");
+
+        var binding1 = actionMap.AddBinding("<Keyboard>/a", firstActionInMap);  // first, 0
+        actionMap.AddBinding("<Keyboard>/b", secondActionInMap); // second, 0
+        actionMap.AddBinding("<Keyboard>/c", firstActionInMap);  // first, 1
+        actionMap.AddBinding("<Keyboard>/d", secondActionInMap); // second, 1
+        actionMap.AddBinding("<Keyboard>/e", secondActionInMap); // second, 2
+
+        var accessor = secondActionInMap.ChangeBinding(binding1.binding.name);
+        Assert.IsTrue(accessor.valid);
+        secondActionInMap.ChangeBinding(new InputBinding("<Keyboard>/e")).WithPath("<Keyboard>/f");
+        Assert.AreEqual(2, actionMap.FindBinding(new InputBinding("<Keyboard>/f"), out _)); // exact match
```

Comment:
Recommend use of `Assert.That` for consistency. Used for all existing tests.

Reply:
Good point, forgot that this was the pattern used here. This have now been updated.